bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_rdataframe.py

```diff
@@ -74,16 +88,77 @@ def RDataFrameAsNumpy(df, columns=None, exclude=None):
     return py_arrays
 
 
+def _histo_profile(self, fixed_args, *args):
+    # Check wheter the user called one of the HistoXD or ProfileXD methods
+    # of RDataFrame with a tuple as first argument; in that case,
+    # extract the tuple items to construct a model object and call the
+    # original implementation of the method with that object.
+
+    # Parameters:
+    # self: instantiation of RDataFrame
+    # fixed_args: list containing e.g. '_OriginalHisto1D' and
+    # e.g. the method Histo1D itself
+    # args: arguments passed by the user when he calls e.g Histo1D
+
+    original_method_name, model_class = fixed_args
+
+    # Get the "original" method of the RDataFrame instantiation
+    original_method = getattr(self, original_method_name)
+
+    if args and isinstance(args[0], tuple):
+        # Construct the model with the elements of the tuple
+        # as arguments
+        model = model_class(*args[0])
+        # Call the original implementation of the method
+        # with the model as first argument
+        if len(args) > 1:
+            res = original_method(model, *args[1:])
+        else:
+            # Covers the case of the overloads with only model passed
+            # as argument
+            res = original_method(model)
+    # Fall back to the original implementation for the rest of the overloads
```

Comment:
`If the first argument is not a tuple, nothing to do, just call the original implementation`.